site/community/release_guide.md

```diff
@@ -0,0 +1,475 @@
+
+    Thanks,
+    Release Manager
+
+    [1] link
+    [2] link
+    [3] link
+    [4] link
+    [5] link
+
+If there are any issues found in the release candidate, reply on the vote thread to cancel the vote. There’s no need to wait 72 hours. Proceed to the `Fix Issues` step below and address the problem. However, some issues don’t require cancellation. For example, if an issue is found in the website pull request, just correct it on the spot and the vote can continue as-is.
+
+If there are no issues, reply on the vote thread to close the voting. Then, tally the votes in a separate email. Here’s an email template; please adjust as you see fit.
+
+    From: Release Manager
+    To: dev@bookkeeper.incubator.apache.org
+    Subject: [RESULT] [VOTE] Release 0.4.0, release candidate #0
+
+    I'm happy to announce that we have unanimously approved this release.
+
+    There are XXX approving votes, XXX of which are binding:
+    * approver 1
+    * approver 2
+    * approver 3
+    * approver 4
+
+    There are no disapproving votes.
+
+    Thanks everyone!
+
+
+### Checklist to proceed to the finalization step
+
+1. Community votes to release the proposed candidate
+
+**********
+
+## Fix any issues
+
+Any issues identified during the community review and vote should be fixed in this step.
+
+Code changes should be proposed as standard pull requests to the `master` branch and reviewed using the normal contributing process. Then, relevant changes should be cherry-picked into the release branch. The cherry-pick commits should then be proposed as the pull requests against the release branch, again reviewed and merged using the normal contributing process.
+
+Once all issues have been resolved, you should go back and build a new release candidate with these changes.
+
+### Checklist to proceed to the next step
+
+1. Issues identified during vote have been resolved, with fixes committed to the release branch.
+
+**********
+
+## Finalize the release
+
+Once the release candidate has been reviewed and approved by the community, the release should be finalized. This involves the final deployment of the release candidate to the release repositories, merging of the website changes, etc.
+
+### Deploy artifacts to Maven Central Repository
+
+Use the Apache Nexus repository to release the staged binary artifacts to the Maven Central repository. In the `Staging Repositories` section, find the relevant release candidate `orgapachebookkeeper-XXX` entry and click `Release`. Drop all other release candidates that are not being released.
+
+#### Deploy source release to dist.apache.org
+
+Copy the source release from the `dev` repository to the `release` repository at `dist.apache.org` using Subversion.
+
+    svn move https://dist.apache.org/repos/dist/dev/bookkeeper/bookkeeper-${VERSION}-rc${RC_NUM} https://dist.apache.org/repos/dist/release/bookkeeper/bookkeeper-${VERSION}
+
+### Update Website
+
+[TBD]
```

Comment:
- in your master branch, go to `site` directory, run `./scripts/release.sh`. it will copy the current documentation for 4.5, bump the versions. these changes are made locally, you need to send a pull request `release x.yz. - website update`. get a review and merge it.
- after this change is merged, merge the release notes pull request.